src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Models/KeyValueSelector.cs

```diff
@@ -52,9 +62,33 @@ public override bool Equals(object obj)
         /// <returns>A hash code for the current object.</returns>
         public override int GetHashCode()
         {
+            int tagsFilterHash = 3;
+
+            if (TagsFilter != null && TagsFilter.Any())
+            {
+                var sortedTags = new SortedSet<string>(TagsFilter);
+
+                if (sortedTags.Any())
+                {
+                    // Concatenate tags into a single string with a delimiter
+                    string tagsString = string.Join("|", sortedTags);
+
+                    // Use SHA256 to generate a hash for the tags
+                    using (var sha256 = System.Security.Cryptography.SHA256.Create())
```

Comment:
Crypto hash is an overkill here. Can we get the hash of each tag filter (string) instead of using SHA256?

Reply:
Oh yea that makes more sense here, thanks